docs/examples/use_cases/tensorflow/efficientdet/dataset/create_tfrecord_indexes.py

```diff
@@ -15,7 +15,7 @@
 r"""Generate TFRecord index files necessary when using DALI preprocessing.
 
 Example usage:
-    python create_tfrecord_indexes.py  --dali_dir=~/DALI  \
+    python create_tfrecord_indexes.py  --tfrecord2idx_script=~/DALI/tools/tfrecord2idx  \
```

Comment:
What's the benefit of this change?

Reply:
I'd see a benefit when the path to `tfrecord2idx` script changes its path inside DALI repo. We'd get a clear error, saying `"{FLAGS.tfrecord2idx_script} does not lead to valid tfrecord2idx script."` instead a vague `"{FLAGS.dali_dir} does not lead to valid DALI installation."`

Reply:
Also, with that change users can use that script even if they installed DALI via pip and don't have DALI source code locally.

Reply:
Ooookaaaay. It's a minor thing and I will not stop this PR over this, but... I still think that we can combine better error message with ease of use (like `Cannot find the tfrecord2idx tool in DALI installation directory. The expected path "{FLAGS.dali_dir}/tools/tfrecord2idx" is invalid. Please check your DALI installation.`). The argument about the file being moved is invalid, because we ship _this_ file with DALI and we still need to keep it up to date (at least to have a proper default). On the other hand, if the user updates DALI _and_ we move the file, then the usage of this script changes - while it wouldn't change if the user updates DALI _and_ examples - the script would pick up the tool from where it should.